extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -162,10 +166,21 @@ export class AmpAnalytics extends AMP.BaseElement {
   layoutCallback() {
     // Now that we are rendered, stop rendering the element to reduce
     // resource consumption.
+    this.element.getResources().setOwner(this.element,
```

Comment:
I'm actually not aware of this behavior. Did you get the call stack and see what logic triggered unlayout?